src/org-unit-select/common.js

```diff
@@ -18,15 +19,17 @@ const style = {
 style.button1 = Object.assign({}, style.button, { marginLeft: 0 });
 
 function addToSelection(orgUnits) {
-    const orgUnitArray = Array.isArray(orgUnits) ? orgUnits : orgUnits.toArray();
-    const addedOus = orgUnitArray.filter(ou => !this.props.selected.includes(ou.path));
+    const { selectableIds, selected } = this.props;
+    const additions = orgUnits.filter(({ id }) =>
+        selectableIds ? selectableIds.includes(id) : true
```

Comment:
Equivalent, but it's perfectly fine as it is: `!selectableIds || selectableIds.includes(id)`

Reply:
Done!